analyzers/its/expected/akka.net/S1144-Akka-netstandard2.0.json

```diff
@@ -54,18 +54,6 @@
       "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka/Event/DeadLetterListener.cs#L255",
       "Location": "Line 255 Position 29-37"
     },
-    {
-      "Id": "S1144",
```

Comment:
Both are FPs: `private readonly` fields inside a type marked with the `StructLayout` attribute.